providers/shared/components/ddb/id.ftl

```diff
@@ -0,0 +1,311 @@
+                        "AttributeSet" : SECRETSTRING_ATTRIBUTESET_TYPE
+                    }
+                    {
+                        "Names" : "Link",
+                        "Description" : "A link to a secret or store that will keep the secret",
+                        "AttributeSet" : LINK_ATTRIBUTESET_TYPE
+                    }
+                ]
+            },
+            {
+                "Names" : "rootCredential:Settings",
+                "Description" : "Store the credentials as settings of the database occurrence",
+                "Children" : [
+                    {
+                        "Names" : "UsernameAttribute",
+                        "Description" : "The setting attribute that contains the username",
+                        "Types" : STRING_TYPE,
+                        "Default" : "MASTER_USERNAME"
+                    },
+                    {
+                        "Names" : "PasswordAttribute",
+                        "Description" : "The setting attribute that contains the password",
+                        "Types" : STRING_TYPE,
+                        "Default" : "MASTER_PASSWORD"
+                    }
+                ]
+            },
+            {
+                "Names" : "Backup",
+                "Children" : [
+                    {
+                        "Names" : "RetentionPeriod",
+                        "Types" : NUMBER_TYPE,
+                        "Default" : 35
+                    },
+                    {
+                        "Names" : "SnapshotOnDeploy",
+                        "Types" : BOOLEAN_TYPE,
+                        "Default" : true
+                    },
+                    {
+                        "Names" : "DeleteAutoBackups",
+                        "Types" : BOOLEAN_TYPE,
+                        "Description" : "Delete automated snapshots when the instance is deleted",
+                        "Default" : true
+                    },
+                    {
+                        "Names" : "DeletionPolicy",
+                        "Types" : STRING_TYPE,
+                        "Values" : [ "Snapshot", "Delete", "Retain" ],
+                        "Default" : "Snapshot"
+                    },
+                    {
+                        "Names" : "UpdateReplacePolicy",
+                        "Types" : STRING_TYPE,
+                        "Values" : [ "Snapshot", "Delete", "Retain" ],
+                        "Default" : "Snapshot"
+                    }
+                ]
+            },
+            {
+                "Names" : "MaintenanceWindow",
+                "AttributeSet" : MAINTENANCEWINDOW_ATTRIBUTESET_TYPE
+            },
+            {
+                "Names" : "BackupWindow",
+                "AttributeSet" : BACKUPWINDOW_ATTRIBUTESET_TYPE
+            },
+            {
+                "Names" : "AutoMinorVersionUpgrade",
+                "Types" : BOOLEAN_TYPE
+            },
```

Comment:
Based on what is stated above this should be removed

Reply:
Done